groupDetails/function.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Maybe change the name, At first I didn't fully understand the purpose of the function